core/src/traits/Kokkos_LoopUnrollTrait.hpp

```diff
@@ -0,0 +1,86 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOS_KOKKOS_LOOPUNROLLTRAIT_HPP
+#define KOKKOS_KOKKOS_LOOPUNROLLTRAIT_HPP
+
+#include <Kokkos_Macros.hpp>
+#include <traits/Kokkos_PolicyTraitAdaptor.hpp>
+#include <traits/Kokkos_Traits_fwd.hpp>
+
+namespace Kokkos {
+
+namespace Experimental {
```

Comment:
Prefer a nested namespace definition
```suggestion
namespace Kokkos::Experimental {
```